ingest/src/main/java/com/microsoft/azure/kusto/ingest/resources/RankedStorageAccount.java

```diff
@@ -0,0 +1,22 @@
+package com.microsoft.azure.kusto.ingest.resources;
+
+public class RankedStorageAccount {
+    private final String accountName;
+
+    public RankedStorageAccount(String accountName) {
+        this.accountName = accountName;
+    }
+
+    public void addResult(boolean success) {
+
```

Comment:
Was this mistakenly unfinished?

Reply:
This PR is just about setting the pieces, since we didn't decide the logic yet.

For now, the shuffling of accounts is completely random.

The idea is to decide on an alogirthm in the future without needing to redo the code.

Reply:
Maybe add a comment to this effect, though.